src/assets/_sass/_dds.scss

```diff
@@ -426,7 +426,7 @@ body {
     width: 100%;
     height: 643px;
     text-align: center;
-    background: url('../img/layout/team-lg.jpg') center top no-repeat;
```

Comment:
delete these if they're not used anywhere

Reply:
✔️